src/communication/azext_communication/aaz/latest/communication/_create.py

```diff
@@ -0,0 +1,386 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "communication create",
+)
+class Create(AAZCommand):
+    """Create a new CommunicationService or update an existing CommunicationService.
+    """
+
+    _aaz_info = {
+        "version": "2023-04-01-preview",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.communication/communicationservices/{}", "2023-04-01-preview"],
```

Comment:
If we're dependent on a preview, should we be merging this into the main branch?

Reply:
@kairu-ms Could you help me answering this question? I don't know what's the process for changes based off preview versions.

Reply:
Yes, the preview version api can be merged into main branch. But the extension version number should be in preview. You can reference the doc above.